.circleci/config.yml

```diff
@@ -1,6 +1,7 @@
 version: 2.1
 setup: true
 orbs:
+  prodsec: snyk/prodsec-orb@1.0
```

Comment:
The secrets scanning is already on master - this is most likely superfluous, once master is merged into develop (develop seems to not have received a merge back from master, yet).